include/game.h

```diff
@@ -774,13 +775,13 @@ typedef enum {
 // the previous enum set. Hacky, but works.
 typedef enum {
     FAM_ACTIVE_NONE,
-    FAM_ACTIVE_BAT,
-    FAM_ACTIVE_GHOST,
-    FAM_ACTIVE_FAERIE,
-    FAM_ACTIVE_DEMON,
-    FAM_ACTIVE_SWORD,
-    FAM_ACTIVE_YOUSEI,     // JP only
-    FAM_ACTIVE_NOSE_DEMON, // JP only
+    FAM_ACTIVE_BAT = FAM_STATS_BAT + 1,
```

Comment:
I'm not sure this is actually cleaner or more moddable but it's not that big of a deal